src/libraries/System.Net.Primitives/src/System/Net/CookieContainer.cs

```diff
@@ -230,7 +230,7 @@ public void Add(Cookie cookie)
             {
                 throw new ArgumentException(
                     SR.Format(SR.net_emptystringcall, nameof(cookie) + "." + nameof(cookie.Domain)),
-                    nameof(cookie) + "." + nameof(cookie.Domain));
+                    nameof(cookie));
```

Comment:
Another case where it seems like we're doing the wrong thing by warning on this case in the analyzer.

Reply:
I think it is fine as `nameof(cookie) + "." + nameof(cookie.Domain)` added to the message already

Reply:
Ok